app/views/install/compose.phtml

```diff
@@ -512,25 +516,25 @@ services:
     command: 'mysqld --innodb-flush-method=fsync'
 
   redis:
-    image: redis:6.0-alpine3.12
+    image: redis:6.2-alpine
```

Comment:
We decided back then to specify the alpine version - since the base image is re-used across all images. So lets try to stick with it 👍🏻 

Not sure if we are consistent, if not or there is no 3.12 for redis 6.2 you can leave it as it is 🙂

Reply:
Leaving as is because there isnt 3.12 version, only 3.13.